components/pages/HomePage/TopSection.tsx

```diff
@@ -110,10 +110,10 @@ export const TopSection: React.ElementType = ({ style }) => {
         >
           <View style={{ flexDirection: "row", alignItems: "center" }}>
             <SmallSection
-              url="/category/magazine/"
+              url="https://issuu.com/thestanforddaily"
```

Comment:
This link can also be refactored into the constants file

Reply:
Done